extra/github-actions/workflows/main.yml

```diff
@@ -0,0 +1,179 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  windows64-build:
+    runs-on: windows-latest
+    # if: false
+    env:
+      PLATFORM: windows64
+      OPAMYES: 1
+      CYG_MIRROR: http://mirrors.kernel.org/sourceware/cygwin/
+      ARCH: 64
+      MINGW_ARCH: x86_64
+      CYGWIN_SETUP: https://cygwin.com/setup-x86_64.exe
+      CYG_ROOT: C:/cygwin64
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+
+      @import install-neko.yml
+      @import build-windows.yml
+
+  windows-build:
+    runs-on: windows-latest
+    # if: false
+    env:
+      PLATFORM: windows
+      OPAMYES: 1
+      CYG_MIRROR: http://mirrors.kernel.org/sourceware/cygwin/
+      ARCH: 32
+      MINGW_ARCH: i686
+      CYGWIN_SETUP: https://cygwin.com/setup-x86.exe
+      CYG_ROOT: C:/cygwin
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+
+      @import install-neko.yml
+      @import build-windows.yml
+
+  linux-build:
+    runs-on: ubuntu-latest
+    # if: false
+    env:
+      PLATFORM: linux64
+      OPAMYES: 1
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+
+      @import build-linux.yml
+
+  mac-build:
+    runs-on: macos-latest
+    # if: false
+    env:
+      PLATFORM: mac
+      OPAMYES: 1
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+
+      @import install-neko.yml
+      @import build-mac.yml
+
+  windows64-test:
+    needs: windows64-build
+    runs-on: windows-latest
+    env:
+      PLATFORM: windows64
+      TEST: ${{matrix.target}}
+      HXCPP_COMPILE_CACHE: ~/hxcache
+      ARCH: 64
+    strategy:
+      fail-fast: false
+      matrix:
+        # TODO. Lua/flash has never been enabled on our AppVeyor builds.
```

Comment:
Flash is probably not so important anymore, but we could investigate Lua on Windows?